### ISSUE TITLE
[GUI] Add Clear Log Button

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#66]
## User story
1. As a user
2. I want to clear the log
3. So that I can remove previous information and focus on the next steps

## Acceptance criteria
* Button near log textarea removes previous output

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
